minStack.js

```diff
@@ -0,0 +1,54 @@
+// Time Complexity : O(1)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this: Is sorting allowed? 
```

Comment:
No, sorting is not allowed. If you are using sorting, the time complexity changes to O(nlogn).